AnnoyancesFilter/Widgets/sections/widgets.txt

```diff
@@ -2261,7 +2261,9 @@ samlab.ws##iframe[src^="https://yoomoney.ru/"]
 !
 hromadske.ua#$#.DonateModal-overlay { display: none !important; }
 hromadske.ua#$#body { overflow: visible !important; }
-!
+inkorr.com##.sub-tg-wrapper
+inkorr.com##.views-row:has(.fa-telegram)
```

Comment:
Should be located in `AnnoyancesFilter/Popups/sections/subscriptions_specific.txt`.
Also, please make more specific like `.view-content .views-row:has(.fa-telegram)`.